### ISSUE TITLE
Lip color again

### ISSUE THREAD

**Issue by akirajay** [sczhou/CodeFormer#121]
#79 reported it as well, i found it is true. 

![takakuraken_before](https://user-images.githubusercontent.com/13637666/214308047-a8367fcd-94d9-4382-b480-4253b5822beb.png)

**Comment by sczhou** [comment]
Did you pull the latest update for the grayscale input? I guess it could fix the lip color for this case.

**Comment by obaidul4** [comment]
Bkhk

**Comment by SkylerZheng** [comment]
I get this too, purple lips and eye color strange too, yellow-brown eyes. Anyone knows how to fix this?

**Comment by finley0066** [comment]
Yes, I get this too, purple lips and eye color strange too, yellow-brown eyes. Anyone knows how to fix this?

**Comment by jackmvision** [comment]
Same here. The color on the lip is incorrect. Anyone got a solution? Thanks!

**Comment by kennethxu** [comment]
It's an old post but OP's image has a yellowish background color so CodeFormer likely threating it as a color image. You can check if the output as a message "Grayscale input: True", your image is considered color unless you see that massage.

A workaround to your problem is to use any photo editor, or tools like ImageMagick/GraphicsMagick to convert the image into a real grayscale image, then run the CodeFormer again.

A better fix is to have CodeFormer accept a parameter to force --grayscale

**Comment by YunanwuNancy** [comment]
How about the color image? I found the lip color changes to a wired color (usually blue) after restoration.
Anyone has any good suggestion and know why?

Thanks.

**Comment by YunanWu2168** [comment]
How about the color image? I found the lip color changes to a wired color (usually blue) after restoration.
Anyone has any good suggestion and know why?

Thanks a lot.
